docs/janssen-server/fido/passkey-telemetry.md

```
@@ -214,6 +222,132 @@ is given below.
+
+```sql
+SELECT "jansAggregationType", "jansId", "jansStartTime"
+FROM "jansFido2MetricsAggregation"
+WHERE "jansAggregationType" = 'HOURLY'
+ORDER BY "jansStartTime" DESC LIMIT 5;
+```
+
+A row should exist for the hour in which you registered. To avoid waiting, query the raw
+entries through the API instead:
+
+```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)